Dockerfile

```diff
@@ -1,3 +1,6 @@
 # Builds a Docker to deliver dist/
 FROM nginx:latest
-COPY dist/ /usr/share/nginx/html
+USER root
```

Comment:
I don't know where this Dockerfile is being used. If it is for testing, then this is fine, but if it is running in production I would advice to not run nginx as root.

Reply:
I think on this repo is it used for CICO. @kbsingh can you confirm?

Reply:
@joshuawilson I don't think so. See https://github.com/fabric8-ui/fabric8-ui/search?utf8=%E2%9C%93&q=Dockerfile&type= . We only use the Dockerfile.builder and Dockerfile.deploy.

Reply:
Like in the other dockerfile, the user can be switched back (and should be IMO, I'm unsure whether to use env_var like the other Dockerfile (Dockerfile.deploy) or the username itself.

The reason to use root is to have enough priv to remove and write content to `/usr/share/nginx/`

Please let me know what you guys think of it and I'll update the PR.

Reply:
@deb please tell us where the Dockerfile is used first before we answer your questions.

Reply:
@deb ah, we have a docker-compose file that uses it: https://github.com/fabric8-ui/fabric8-ui/blob/master/docker-compose.yaml#L15